// // Exercice 1:  Transformer une exécution en Result (fromExecution),  puis utiliser match pour afficher le résultat ou une erreur

// const riskyFunction = () => {
//   if (Math.random() > 0.5) {
//     return 100;
//   }
//   throw new Error('Erreur');
// };
export const resultFn1 = () => {};
